PATCH-P0-HACK-UPSTREAM: Add lib64 dir to the library path

On x86_64 systems openssl library is installed into /usr/lib64 directory and
MySQL couldn't find as it was searching only in /usr/lib. We can add it quite
safelly even on i586 as this directory wouldn't exist on them so no harm done.

Maintainer: Michal Hrusecky <Michal.Hrusecky@opensuse.org>

--- config/ac-macros/ssl.m4	2009-04-10 14:05:07.000000000 +0200
+++ config/ac-macros/ssl.m4	2009-04-10 14:15:15.000000000 +0200
@@ -102,7 +102,7 @@
   #
   # Try to link with openSSL libs in <location>
   #
-  openssl_libs="-L$location/lib/ -lssl -lcrypto"
+  openssl_libs="-L$location/lib -L$location/lib64 -lssl -lcrypto"
   MYSQL_CHECK_SSL_DIR([$openssl_includes], [$openssl_libs])
 
   if test "$mysql_ssl_found" == "no"
